sdks/python/apache_beam/io/fileio.py

```diff
@@ -649,7 +649,7 @@ class _CompressedFile(object):
   def __init__(self,
                fileobj,
                compression_type=CompressionTypes.GZIP,
-               read_size=16384):
+               read_size=gcsio.DEFAULT_READ_BUFFER_SIZE):
```

Comment:
Created https://issues.apache.org/jira/browse/BEAM-1222 to remove GCS specific constants from fileio.